contrib/gce/configure.sh

```diff
@@ -235,6 +238,22 @@ disabled_plugins = ["io.containerd.internal.v1.restart"]
 EOF
 chmod 644 "${config_path}"
 
+
+docker_registry_host_namespace="${registry_config_path}/docker.io/hosts.toml"
+mkdir -p $(dirname ${docker_registry_host_namespace})
+cat > ${config_path} <<EOF
+server = "https://docker.io"
```

Comment:
Confusingly still, `server = "https://registry-1.docker.io"`, docker.io is not a valid registry server today.